pkg/descheduler/evictions/evictions.go

```diff
@@ -562,7 +564,7 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 }
 
 // return (ignore, err)
-func (pe *PodEvictor) evictPod(ctx context.Context, pod *v1.Pod) (bool, error) {
+func (pe *PodEvictor) evictPod(ctx context.Context, pod *v1.Pod, policyGroupVersion string, gracePeriodSeconds *int64) (bool, error) {
 	deleteOptions := &metav1.DeleteOptions{}
```

Comment:
```suggestion
func (pe *PodEvictor) evictPod(ctx context.Context, pod *v1.Pod) (bool, error) {
	deleteOptions := &metav1.DeleteOptions{
		GracePeriodSeconds: pe.gracePeriodSeconds,
	}
```

Reply:
Awkward..... The changes here were undone during rebase. thanks for this